src/tests/multihost/ad/test_adparameters.py

```diff
@@ -705,25 +707,21 @@ def test_0018_bz1734040(self, multihost, adjoin):
         domain_name = client.get_domain_section_name()
```

Comment:
unused, can be removed or used on line 710

Reply:
Fixed.